COLLABORATOR_GUIDE.md

```diff
@@ -473,16 +473,20 @@ level.
 
 ### Involving the TSC
 
-Collaborators may opt to elevate pull requests or issues to the [TSC][] for
-discussion by assigning the `tsc-review` label or @-mentioning the
-`@nodejs/tsc` GitHub team. This should be done where a pull request:
+Collaborators may opt to elevate pull requests or issues to the [TSC][].
+This should be done where a pull request:
 
 - is labeled `semver-major`, or
 - has a significant impact on the codebase, or
 - is inherently controversial, or
 - has failed to reach consensus amongst the Collaborators who are
   actively participating in the discussion.
 
+Assign the `tsc-review` label or @-mention the
+`@nodejs/tsc` GitHub team if you want to elevate an issue to the [TSC][].
+Do not use the GitHub UI on the right hand side to assign to`
```

Comment:
Rather than making this a hard rule without documenting the reasons for it, consider rewording as a suggestion with the reasons why one might want to avoid it. Also, consider, that the same likely applies to requesting review from *any* team, not just the `@nodejs/tsc`.

The following is too wordy given that it's 6:30am on a Saturday and I haven't had enough coffee but this should illustrate the point:

```
Use of the GitHub UI to assign an issue to, or request a review
from the `@nodejs/tsc` (or any team) is not recommended because,
at the current time, doing so causes unnecessary notifications to
be sent to team members who may already have provided feedback or
reviews for the issue or PR. It is better to selectively request
reviews on an individual basis.
```

More Friendly-Advice-To-Make-Co-Contributors-Lives-Easier, less Yet-Another-Rule-One-Has-To-Follow.